bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```
@@ -0,0 +1,351 @@
+thing-type.netatmo.doorbell.label = Smart Video Doorbell
+thing-type.netatmo.doorbell.description = The Netatmo Smart Video Doorbell device.
+thing-type.netatmo.home-coach.label = Healthy Home Coach
+thing-type.netatmo.home-coach.description = Healthy home coach reporting health-index, temperature, humidity, pressure, air quality and sound level.
+thing-type.netatmo.home.label = Home
+thing-type.netatmo.home.description = A home hosting Security or Energy devices and modules.
+thing-type.netatmo.indoor.label = Additional Module
+thing-type.netatmo.indoor.description = Additional indoor module reporting temperature, humidity and CO2 level.
+thing-type.netatmo.outdoor.label = Outdoor Module
+thing-type.netatmo.outdoor.description = Outdoor module reporting temperature and humidity.
+thing-type.netatmo.person.label = Person
+thing-type.netatmo.person.description = A person known by your Netatmo system.
+thing-type.netatmo.plug.label = Relay Plug
+thing-type.netatmo.plug.description = The relay connected to the boiler controlling a Thermostat and zero or more valves.
+thing-type.netatmo.presence.label = Smart Outdoor Camera
+thing-type.netatmo.presence.description = The Netatmo Smart Outdoor Camera (Presence) camera with or without siren.
+thing-type.netatmo.rain.label = Rain Gauge
+thing-type.netatmo.rain.description = Rain Gauge measuring precipitation.
+thing-type.netatmo.room.label = Room
+thing-type.netatmo.room.description = A room in your house.
+thing-type.netatmo.siren.label = Siren Module
+thing-type.netatmo.siren.description = The Netatmo Smart Indoor Siren.
+thing-type.netatmo.thermostat.label = Thermostat Module
+thing-type.netatmo.thermostat.description = The Thermostat device placed in a given room.
+thing-type.netatmo.valve.label = Radiator valve
+thing-type.netatmo.valve.description = A valve controlling a radiator.
+thing-type.netatmo.weather-station.label = Weather Station
+thing-type.netatmo.weather-station.description = Main indoor module reporting temperature, humidity, pressure, air quality and sound level.
+thing-type.netatmo.welcome.label = Smart Indoor Camera
+thing-type.netatmo.welcome.description = The Netatmo Smart Indoor Camera (Welcome).
+thing-type.netatmo.wind.label = Wind Gauge Module
+thing-type.netatmo.wind.description = Wind sensor reporting wind angle and strength.
+
+# error messages
+
+conf-error-no-client-id = Cannot connect to Netatmo bridge as no client id is available in the configuration
+conf-error-no-client-secret = Cannot connect to Netatmo bridge as no client secret is available in the configuration
+conf-error-no-username = Cannot connect to Netatmo bridge as no username is available in the configuration
+conf-error-no-password = Cannot connect to Netatmo bridge as no password is available in the configuration
+status-bridge-offline = Bridge is not connected to Netatmo API
+device-not-connected = Device is not connected
```

Quite minor, but could this be rephrased to:
```suggestion
device-not-connected = Device is not reachable
```
or is it used in more than this exact scenario? Is it always a module (additional indoor/outdoor)? In this case it could even be "Module is not reachable".

---

Done

---

I should wait for a new submit?

---

You just received it !